web/vote.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+	require_once('config.php');
+	if(!isset($_COOKIE['retainVote'])) {
+		setcookie('retainVote', 1, time()+60, '/', $domain);
```

Comment:
Is 60 seconds long enough? The page might be open for quite a while, for multiple polls.